jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
+        self.spin_counter = 50
+        self.current_round = 1
+        self.turnPoints = 0
+        self.question_bank
+
+    'Add a player to the list'
+    def addPlayer(self, name):
```

Comment:
I see what you are doing here but ultimately we may change this so the class inherits an array of Player classes. For now this should be fine!